collections/pixel-art-tools/index.md

```diff
@@ -9,6 +9,7 @@ items:
  - https://github.com/jennschiffer/make8bitart
  - https://github.com/kitao/pyxel
  - https://github.com/jackschaedler/goya
+ - https://github.com/cloudhead/rx
```

Comment:
I've updated the syntax of this file upstream. Can you update this PR with the new syntax?

We can now use:
```md
  - cloudhead/rx
```

Reply:
Yes will, do, but I will have to do it in a different PR, as I accidentally deleted my fork 😓